config/lint/lint.xml

```diff
@@ -79,4 +79,6 @@
 
     <!-- TODO: https://github.com/wordpress-mobile/WordPress-Android/issues/21065 -->
     <issue id="UsingMaterialAndMaterial3Libraries" severity="warning" />
+    <!--  We rely on more granural checks from WordPress-Lint-Android  -->
+    <issue id="UnknownNullness" severity="ignore" />
```

Comment:
Suggestion (💡): Instead of ignoring `UnknownNullness` I actually recommend we revert #19116 and remove all the configuration related to that, effectively removing it, not ignoring. 🤔

Reply:
Thank you @ParaskP7 ! I've applied your suggestion and rebranded this PR to not introduce changed WordPress Lint checks. I don't have capacity to work on this now, but I think PR in this shape will also be useful to merge!